### DISCUSSION TITLE
additional_payer_notes is not required in JSON format, but seems to be required in the wide CSV and not valid in the tall format

### DISCUSSION THREAD

**Discussion by nmaylevc** [CMSgov/hospital-price-transparency#125] [Q&A] [answered]
Thie additional_payer_notes is not required in JSON format, but seems to be required in the wide CSV format.
In the table of required fields it has a "--" for the tall format, but this field certainly is allowed for the tall format.

This seems a bit messed up.  

For wide and tall should these be required?  For tall, if any payer-plan has a payer-specific note for any item one would need the field. For wide, if any item has a payer-specific note for this payer-plan, then the field needs to be there.  The other way would just to leave it required.  If it is required, then it should be required for both wide and tall.  That wouldn't be consistent with JSON, but it would be tolerable.  Right now the tall doesn't seem to have this field.

These comments are based on the Required Standard Charge, Item/Service, and Coding Data Elements table on the documentation page.

**Comment by RidenourK** [comment, marked as answer]
@nmaylevc. Thank you for your question. 

Additional notes are required in all three formats in certain scenarios. Please refer to the conditional requirements in the CSV and JSON documentation for more information about when additional notes are required. 

As indicated in the CY 2024 OPPS/ASC Final Rule, the “tall’’ format contains separate payer and plan data elements that provide the benefit of static header naming with less opportunity for standardization error and that is similar to existing output files that many hospitals have been using to build their MRFs. The ‘‘wide’’ format contains variable payer-specific negotiated charge data elements that incorporate the payer and plan name into a single column header, which may reduce the file size because many data elements would not need to be repeated as frequently. Please refer to the discussion at [88 FR 82088](https://www.federalregister.gov/d/2023-24293/p-4893).

Therefore, additional payer-specific notes is intentionally not included as a column header in the CSV Tall format template. Please see column Q in the CSV Wide examples and column U in the CSV Tall examples for examples of how payer-specific notes can be encoded in both formats.
